### ISSUE TITLE
No `location` on output

### ISSUE THREAD

**Issue by adamburmister** [0x8890/docchi#15]
I'm using docchi to parse some Jasmine test files to extract useful JSdoc tags to tie the tests back to their tickets.

I would like to receive the location of each comment block, which the documentation leads me to believe is already done, but I receive no output on location...

Output is:

```js
[ { comment: { description: '<p>Before comment</p>\n', tags: [Object] } },
  { comment:
     { description: '<p>Describe block comment</p>\n',
       tags: [Object] } },
  { comment: { description: '', tags: [Object] } } ]
```

**Comment by 0x8890** [comment]
@adamburmister  What is your input?

**Comment by adamburmister** [comment]
```js
'use strict';

describe('Comments test', () => {

  /**
   * Before comment, no tags
   * @param  {function} done
   * @return {void}
   */
  before(done => done())

  /**
   * Describe block comment
   * @jira PAYXCO-123
   * @testrail 1234
   * @region US
   * @role admin
   */
  describe('describe block comment', () => {

    /**
     * @role admin
     * @region global
     * @jira PAYXCO-123
     * @testrail 1234
     */
    it('should generate an `it` comment', (done) => {
      done();
    });

  });
});
```

**Comment by 0x8890** [comment]
Fixed in commit https://github.com/0x8890/docchi/commit/8c347f3aaa97a43cfd19282eda925f9708133fc2, published as `0.11.0`.

**Comment by adamburmister** [comment]
Thank you!
